src/Control/Monad/Rec/Class.purs

```diff
@@ -20,6 +21,9 @@ import Control.Monad.Error.Trans
 import Control.Monad.State.Trans
 import Control.Monad.Writer.Trans
 
+
+data Step a b = Loop a | Done b
```

Comment:
Could you please add a markdown comment? Just something like "The result of a computation: either `Loop`, updating the accumulator, or indicate that the computation is `Done`."